src/routes/docs/builders/dialog/drawer-code.ignore-svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+	import { createDialog } from '@melt-ui/svelte';
+	import { fade } from 'svelte/transition';
+	import { expoOut } from 'svelte/easing';
+	import X from 'icons';
+
+	const { trigger, portal, overlay, content, title, description, close, open } = createDialog();
+
+	function slide(node: HTMLElement, options: { x: number; duration: number }) {
+		return {
+			duration: options.duration,
+			delay: 0,
+			css: (t: number) => {
+				const p = t * (0 - options.x) + options.x;
+				return `transform: translateX(${p}px)`;
+			},
+			easing: expoOut,
+		};
+	}
+</script>
+
+<div>
+	<button
+		{...$trigger()}
+		class="inline-flex items-center justify-center rounded-md bg-white px-4 py-2
+			font-medium leading-none text-magnum-700 shadow-lg hover:opacity-75
+			focus:outline-none focus:ring focus:ring-magnum-400"
+	>
+		Open Drawer
+	</button>
+	<div use:portal>
+		{#if $open}
+			<div
+				{...$overlay}
+				class="fixed inset-0 z-20 bg-black/50"
+				transition:fade|local={{ duration: 150 }}
+			/>
+			<div
+				transition:slide|local={{ x: -350, duration: 200 }}
+				class="fixed left-0 top-0 z-[50] h-screen w-full max-w-[350px] bg-white p-[25px] shadow-lg
```

Comment:
`z-50` is built in so you don't need to use an arbitrary value

Reply:
Thanks!